src/reporting.jl

```diff
@@ -34,7 +36,7 @@ Return a reporter which can be used for progress reports with a known number of
 `total_steps`. May return the same reporter, or a related object. Will display `meta` as
```

Comment:
please document currently_warmup in the docstring

Reply:
Done!